$OpenBSD: patch-RunGrepTest,v 1.4 2020/06/20 06:55:46 kn Exp $

Our sed(1) doesn't cope with NUL bytes and \x00-style notation.

Index: RunGrepTest
--- RunGrepTest.orig
+++ RunGrepTest
@@ -740,9 +740,9 @@ $valgrind $vjs $pcre2grep -n --newline=anycrlf "^(abc|
 printf '%c--------------------------- Test N7 ------------------------------\r\n' - >>testtrygrep
 uname=`uname`
 case $uname in
-  Linux)
+  OpenBSD)
     printf 'abc\0def' >testNinputgrep
-    $valgrind $vjs $pcre2grep -na --newline=nul "^(abc|def)" testNinputgrep | sed 's/\x00/ZERO/g' >>testtrygrep
+    $valgrind $vjs $pcre2grep -na --newline=nul "^(abc|def)" testNinputgrep | gsed 's/\x00/ZERO/g' >>testtrygrep
     echo "" >>testtrygrep
     ;;
   *)
